### ISSUE TITLE
[bug?] naemon does logging to /var/log/messages

### ISSUE THREAD

**Issue by beinbliss** [naemon/naemon#53]
Even if use_syslog=0 in /omd/sites/site/etc/naemon/naemon.d/logging.cfg
still it logs to /var/log/messages:

...
Mar 22 15:28:14 changed tag_naemon_log: [1458653294] HOST ALERT: changed;UP;HARD;1;PING OK - Packet loss = 0%, RTA = 0.54 ms
Mar 22 15:28:14 changed tag_naemon_log: [1458653294] HOST ALERT: changed;UP;HARD;1;PING OK - Packet loss = 0%, RTA = 0.74 ms
...

OS: Centos 7
Package: omd-2.11.20160318-labs-edition-el7-38.x86_64

**Comment by beinbliss** [comment]
nevermind. local issue with syslog
